housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/validate/MissingTableValidationFailure.java

```diff
@@ -15,31 +15,19 @@
  */
 package com.hotels.housekeeping.tool.vacuum.validate;
 
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.io.Writer;
-
 import com.hotels.housekeeping.tool.vacuum.conf.Table;
 
-class UnexpectedValidationFailure implements ValidationFailure {
+class MissingTableValidationFailure implements ValidationFailure {
 
   private final Table table;
-  private final Exception e;
 
-  UnexpectedValidationFailure(Table table, Exception e) {
+  MissingTableValidationFailure(Table table) {
     this.table = table;
-    this.e = e;
   }
 
   @Override
   public String getMessage() {
-    Writer stringWriter = new StringWriter();
-    try (PrintWriter writer = new PrintWriter(stringWriter)) {
-      writer.println("Failed to validate with exception:");
-      e.printStackTrace(writer);
-      writer.flush();
-    }
-    return stringWriter.toString();
+    return "Table does not exist.";
```

Comment:
Put the table name in the message?

Reply:
Oh, wait, you can get it from the `getQualifiedTableName` method when this gets logged, never mind.

Reply:
yes because I want the class doing the actual logging line it all out the same